experiments/golden-results/Tokeneer-summary.txt

```diff
@@ -1225,17 +1225,17 @@ Error detected at REDACTED
 --
 Occurs: 1 times
 +===========================GNAT BUG DETECTED==============================+
-| GNU Ada (ada2goto) Assert_Failure failed precondition from tree_walk.ads:108|
+| GNU Ada (ada2goto) Assert_Failure range_check.adb:509                    |
```

Comment:
These look like crashes in the introduced code, can you easily see why this is happening?

Reply:
The issue is with system.ads, It was failing in the call to Do_Expression because the node kind was not N_Subexpr.
Now it is trying to do a range check first and the value of the range checked value is not a Class_Expr.